### ISSUE TITLE
Main: i18n export causing cyclic import errors

### ISSUE THREAD

**Issue by deepanchal** [CrisisCleanup/crisiscleanup-4-web#26]
This i18n export shouldn't be exported from `main.ts`

```ts
const getI18n = (messages = {}) => {
  return createI18n({
    legacy: false,
    formatFallbackMessages: true,
    silentFallbackWarn: false,
    locale: 'en',
    messages,
  });
};

export const i18n = getI18n();
```

A better solution would be to use provide/inject here:

Provide `i18n` from `main.ts`

```ts
// ...
const i18n = getI18n();

// ...
app.provide('i18n', i18n);

```

Inject it in hooks/components:

```ts
// ...
const i18n = inject('i18n');
// ...
```

**Comment by aarontitus** [comment]
Braden still needs to review.

**Comment by deepanchal** [comment]
Fixed by #540
